pkgs/development/python-modules/eyed3/default.nix

```diff
@@ -30,8 +30,7 @@ buildPythonPackage rec {
 
   postInstall = ''
     for prog in "$out/bin/"*; do
-      wrapProgram "$prog" --prefix PYTHONPATH : "$PYTHONPATH" \
-                          --prefix PATH : ${python}/bin
+      wrapProgram "$prog" --prefix PATH : ${python}/bin
```

Comment:
The whole `wrapProgram` can be removed because `{python}/bin` is wrapped by the builder too

Reply:
you're right, thanks for pointing that out.